recipes/liboptv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: optv
+  version: 0.2.3
+
+source:
+  git_url: https://github.com/OpenPTV/openptv.git
+
+build:
+  number: 1
+  script: cd py_bind && python setup.py prepare && python setup.py install
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - setuptools
+    - numpy x.x
+    - cython
+    - pyyaml
```

Comment:
Please, the libraries which you need to build this project should be in ``host``. Following your requirements, that should be something like:
```
requirements:
  build:
    - {{ compiler('cxx') }}
  host:
    - python
    - setuptools
    - numpy
    - cython
    - pyyaml
  run:
    - {{ pin_compatible('numpy') }}
```